iceoryx_posh/test/moduletests/test_mepoo_handler.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019, 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
Please read the https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md as preparation before contributing and adjust your IDE so that it uses clang-format.

Please use the capital U postfix for all unsigned integer assigments. Example `uint64_t bla = 1U;`